isaac_ros2_control.repos

```diff
@@ -0,0 +1,6 @@
+# Used with vcstool, users can download and source of all packages in this.
+repositories:
+  isaac_ros2_control:
```

Comment:
Why do we have a `.repo` file with only this repo in it? I suggest we remove it till it's needed.

Reply:
We require a custom version of `moveit_resources` maybe we should add it here?

Reply:
I will remove it for now, I'll add the dependencies back in the documentation PR